doc/iterations/2024.5/shapes/configurable_visibility.adoc

```diff
@@ -0,0 +1,47 @@
+= Configurable visibility
```

Comment:
Since the hidden is part of a trio of options (normal, faded, hidden) this topic is a bit more than just hidden / visible, faded must be part of the mix since it will be requested and it's a core part of this topic.